src/main/js/main_html5.js

```diff
@@ -855,6 +855,31 @@ require("../../../html5-common/js/utils/environment.js");
       }
     };
 
+    /**
+     * For multi audio we can get a list of available audio tracks
+     * @public
+     * method OoyalaVideoWrapper#getAvailableAudio
+     * @returns {(boolean|Array)} false - if an array with tracks is not available or Array - an array of all available audio tracks.
+     */
+    this.getAvailableAudio = function() {
+      var audioTrack = _video.audioTracks;
+      if (audioTrack !== undefined && audioTrack.length) {
+        var audioTrackList = [];
+        for (var i = 0; i < audioTrack.length; i++) {
```

Comment:
Minor thing. Can you change i to index or something else more meaningful in another checkin?  As part of our new coding standards, we want to avoid single letter variable names.  In case you missed it: https://docs.google.com/document/d/1VD1QTWO6r1SBR2bPFUTD2CtVD4I_1JyKZnpuXVyxB-U/edit

Reply:
\+